### PR TITLE
use cri-tools version from dependencies.yaml

### PR DESCRIPTION
**Pull request by sohankunkerkar**
Signed-off-by: Sohan Kunkerkar <sohank2602@gmail.com>

#### What type of PR is this?

/kind cleanup

#### Does this PR introduce a user-facing change?

```release-note
None

```

### CODE REVIEW

**Comment by haircommander** [comment]
/approve
/lgtm

thanks!

File: contrib/test/ci/setup.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@
 - name: install Golang tools
   include: "golang.yml"
   vars:
-    version: "1.19"
+    golang_version: "1.19"
```
Comment (saschagrunert) [inline contrib/test/ci/setup.yml:8]: I don't think we need this since we use `{{ golang_version }}` directly in golang.yml.

Comment (sohankunkerkar) [reply contrib/test/ci/setup.yml]: Yup, good catch!

File: contrib/test/ci/main.yml [hunk 2]
```diff
@@ -1,195 +0,0 @@
----
```
Comment (saschagrunert) [inline contrib/test/ci/main.yml:1]: What's the reason that we removed that file?

Comment (sohankunkerkar) [reply contrib/test/ci/main.yml:1]: This file has not been used for running any of our new CI jobs.

**Comment by sohankunkerkar** [comment]
There's a [PR](https://github.com/cri-o/cri-o/pull/6220) to fix the `e2e-conomn-rs` job

**Pull request merged**